packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/form.tsx

```diff
@@ -1,13 +1,13 @@
-import { ThemedScrollbars, Text } from '@deriv/components';
-import { isSafari, isMobile } from '@deriv/shared';
-import { localize } from '@deriv/translations';
+import React from 'react';
 import classNames from 'classnames';
 import { Form, Formik, FormikProps } from 'formik';
-import React from 'react';
 import * as Yup from 'yup';
-import { QuickStrategyFields, QuickStrategyFooter } from '.';
+import { Text, ThemedScrollbars } from '@deriv/components';
+import { isMobile, isSafari } from '@deriv/shared';
+import { localize } from '@deriv/translations';
 import { TQuickStrategyFormValues } from '../quick-strategy.types';
 import { TQuickStrategyForm } from './components.types';
+import { QuickStrategyFields, QuickStrategyFooter } from '.';
```

Comment:
why do we need these changes in bot package ?

Reply:
It is there already just that when i git push, all the imports in other files will change it's sequence

Reply:
I've tried make it to it's default value but when i push the import sequence will go back to this format.